Add SIG Docs Spotlight blog article (#323)
kubernetes/contributor-site: content/en/blog/2022/sig_docs_spotlight_blog.md

---
layout: blog
title: "Spotlight on SIG Docs"

> date: 29-07-2022

Reviewer: Use an ISO8601 date:
```suggestion
date: 2022-07-29
```

Right now, it would render as:
https://k8s.dev/blog/1/01/01/sig-docs-spotlight-2022/

slug: sig-docs-spotlight-2022
---

## Spotlight on SIG-Docs

> The official documentation is the go-to source for any open source project. For Kubernetes, it's an                                                                 ever-evolving Special Interest Group(SIG) with people constantly putting in their efforts to make                                                                details about the project easier to consume for new contributors and users. SIG Docs publishes the official documentation on 

Reviewer: Please fix the weird wrapping here.

Author: I think it's fixed now, pls check

**Blogs**: This subproject highlights new or graduated Kubernetes enhancements, community reports, SIG updates 
or any relevant news to the Kubernetes community such as thought leadership, tutorials and project updates, 
such as the Dockershim removal and removal of PodSecurityPolicy, which is upcoming in the 1.25 release.
Tim Bannister, one of the SIG Docs tech leads, does awesome work and is a major force when pushing contributions 
through to the docs and blogs.

**Localization**: With this subproject, the Kubernetes community has been able to achieve greater inclusivity 
and diversity among both users and contributors. This has also helped the project gain more contributors, 
especially students, since a couple of years ago.
One of the major highlights and up-and-coming localizations are Hindi and Bengali. The efforts for Hindi 
localization are currently being spearheaded by students in India.

In addition to that, there are two other subprojects: [reference-docs](https://github.com/kubernetes-sigs/reference-docs) and the [website](https://github.com/kubernetes/website), which is built with Hugo and is an important ownership area.

> ### Recently there has been a lot of buzz around the Kubernetes ecosystem as well as the industry regarding the removal of Dockershim in the latest 1.24 release. How has SIG Docs’ helped the project to ensure a smooth change among the end-users?

Reviewer: I'd write:
```suggestion
### Recently there has been a lot of buzz around the Kubernetes ecosystem as well as the industry regarding the removal of dockershim in the latest 1.24 release. How has SIG Docs helped the project to ensure a smooth change among the end-users? {#dockershim-removal}
```

---
layout: blog
title: "Spotlight on SIG Docs"
date: 2022-07-29
slug: sig-docs-spotlight-2022
---

> ## Spotlight on SIG-Docs

Reviewer: ```suggestion
```

Not needed, the layout already adds a title `<h1>`.

The official documentation is the go-to source for any open source project. For Kubernetes, 

> it's an ever-evolving Special Interest Group(SIG) with people constantly putting in their efforts 

Reviewer: ```suggestion
it's an ever-evolving Special Interest Group (SIG) with people constantly putting in their efforts
```

to make details about the project easier to consume for new contributors and users. SIG Docs publishes 
the official documentation on [kubernetes.io](https://kubernetes.io) which includes, 
but is not limited to, documentation of the core APIs, core architectural details, and CLI tools 
shipped with the Kubernetes release. 

To learn more about the work of SIG Docs and its future ahead in shaping the community, I have summarised 

> my conversation with the co-chairs, [Divya Mohan](https://twitter.com/Divya_Mohan02)(DM), 
> [Rey Lejano](https://twitter.com/reylejano)(RL) and Natali Vlatko (NV), who walk us through the 

Reviewer: I'd write:
```suggestion
my conversation with the co-chairs, [Divya Mohan](https://twitter.com/Divya_Mohan02) (DM), 
[Rey Lejano](https://twitter.com/reylejano) (RL) and Natali Vlatko (NV), who ran through the
```
